pages/docs/reference/basic-syntax.md

```diff
@@ -602,3 +602,45 @@ fun main(args: Array<String>) {
 </div>
 
 See [Higher-order functions and Lambdas](lambdas.html).
+
+## Creating basic classes and its instances:
+
+<div class="sample" markdown="1">
+
+``` kotlin
+//sampleStart
+fun main(args: Array<String>) {
+    val rectangle: Rectangle = Rectangle (5.0, 2.0) //no 'new' keyword required in Kotlin while initializing
+    val triangle: Triangle = Triangle (3.0, 4.0, 5.0)
+    print( "Area of rectangle is - ${rectangle.getArea()} and it's perimeter is - ${rectangle.getPerimeter()}" )
```

Comment:
Note: the two strings will be printed on the same line. Maybe `println` is better here.

Reply:
I'll remove all syntax errors.
Thanks.